six_patches_of_pain.go

```diff
@@ -359,23 +364,43 @@ func isGNT4(filePath string) bool {
 		if reflect.DeepEqual(expected, data[:len]) {
 			fmt.Println("Validating GNT4 ISO is not modified...")
 			hashValue, err := hashFile(filePath)
+			fmt.Println(hashValue)
 			check(err)
+			// 60aefa3e is the CRC32 hash of both a good ISO dump AND an Nkit ISO somehow
+			// Check whether this file is an ISO or an Nkit
 			if hashValue == "60aefa3e" {
```

Comment:
Somehow, 60aefa3e manages to be the checksum for the clean rip of the ISO **and** the nkit ISO, despite them being different sizes with different bytes. I am shocked that this was a hash collision.